Titanium.Web.Proxy/Helpers/Tcp.cs

```diff
@@ -202,7 +147,7 @@ internal static TcpRow GetTcpRowByLocalPort(IpVersion ipVersion, int localPort)
                     writer.Flush();
 
                     var data = ms.ToArray();
-                    await ms.WriteAsync(data, 0, data.Length);
+                    await clientStream.WriteAsync(data, 0, data.Length);

```

Comment:
This was the problem in #284. The header comes from the client, it should be sent to the server, not to the client.